src/components/Button/Button.module.scss

```
@@ -0,0 +1,16 @@
+@import "@styles/reset.scss";
+@import "@styles/varibles.scss";
+
+.search-button {
+  padding: 14px 16px;
+  background: $accent-color;
+  border-radius: $block-border-radius;
+
+  &_hover:hover {
+    opacity: 0.7;
+  }
+
+  &_disabled:disabled {
+    background: $button-disabled-color;
+  }
```

нет смысла выносить в отдельные классы
```suggestion
  &:hover {
    opacity: 0.7;
  }

  &:disabled {
    background: $button-disabled-color;
  }
```